### PR TITLE
chore: fix spelling errors

### PR DESCRIPTION
**Pull request by jsoref**
## Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/trivy/commit/a47f1f5fe8a5b31ac45cdd71b87ed1e9a608a41c#commitcomment-76469490

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/trivy/commit/59d72f57dcd04cf6ec780f18b14847c3b2900880

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I will be annotating this PR, and I fully expect to be asked to drop certain files.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/community/maintainer/triage.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ We use two labels [help wanted](https://github.com/aquasecurity/trivy/issues?q=i
 and [good first issue](https://github.com/aquasecurity/trivy/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
 to identify issues that have been specially groomed for new contributors.
 
-We have specific [guidelines](/docs/docs/advanced/contribd/contrib/help-wanted.md)
+We have specific [guidelines](/docs/community/maintainer/help-wanted.md)
```
Comment (jsoref) [inline docs/community/maintainer/triage.md:191]: Notable

File: docs/docs/integrations/github-actions.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # GitHub Actions
 
-- Here is the [Trivy Github Action][action]
+- Here is the [Trivy GitHub Action][action]
```
Comment (jsoref) [inline docs/docs/integrations/github-actions.md:3]: Brand

File: docs/docs/integrations/gitlab-ci.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ container_scanning:
 [example]: https://gitlab.com/aquasecurity/trivy-ci-test/pipelines
 [repository]: https://github.com/aquasecurity/trivy-ci-test
 
-### Gitlab CI alternative template
+### GitLab CI alternative template
```
Comment (jsoref) [inline docs/docs/integrations/gitlab-ci.md:114]: Brand

File: docs/index.md [hunk 4]
```diff
@@ -22,17 +22,17 @@ All you need to do for scanning is to specify a target such as an image name of
     <h1 id="demo">Demo</h1>
 </div>
 
-<figure style="text-aligh: center">
+<figure style="text-align: center">
```
Comment (jsoref) [inline docs/index.md:25]: notable

File: helm/trivy/README.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ This chart bootstraps a Trivy deployment on a [Kubernetes](http://kubernetes.io)
 - Kubernetes 1.12+
 - Helm 3+
 
-## Installing from the the Aqua Chart Repository
+## Installing from the Aqua Chart Repository
```
Comment (jsoref) [inline helm/trivy/README.md:21]: This is a fairly recent heuristic, I'm reasonably confident it's correct.

File: integration/client_server_test.go [hunk 6]
```diff
@@ -225,7 +225,7 @@ func TestClientServer(t *testing.T) {
 			golden: "testdata/mariner-1.0.json.golden",
 		},
 		{
-			name: "buxybox with Cargo.lock",
+			name: "busybox with Cargo.lock",
```
Comment (jsoref) [inline integration/client_server_test.go:228]: ?

File: integration/testdata/almalinux-8.json.golden [hunk 7]
```diff
@@ -68,7 +68,7 @@
...
...L terminated, even though this is not guaranteed for strings that have been directly constructed. Where an application requests an ASN.1 structure to be printed, and where that ASN.1 structure contains ASN1_STRINGs that have been directly constructed by the application without NUL terminating the \"data\" field, then a read buffer overrun can occur. The same thing can also occur during name constraints processing of certificates (for example if a certificate has been directly constructed by the application instead of loading it via the OpenSSL parsing functions, and the certificate contains non NUL terminated ASN1_STRING structures). It can also occur in the X509_get1_email(), X509_REQ_get1_email() and X509_get1_ocsp() functions. If a malicious actor can cause an application to directly construct an ASN1_STRING and then process it through one of the affected OpenSSL functions then this issue could be hit. This might result in a crash (causing a Denial of Service attack). It could also result in the disclosure of private memory contents (such as private keys, or sensitive plaintext). Fixed in OpenSSL 1.1.1l (Affected 1.1.1-1.1.1k). Fixed in OpenSSL 1.0.2za (Affected 1.0.2-1.0.2y).",
```
Comment (jsoref) [inline integration/testdata/almalinux-8.json.golden:71]: These CVEs have errors in them. I'm happy to drop the files. If someone knows offhand how to get the CVEs fixed, that'd probably be better for the community overall.

File: integration/testdata/fixtures/db/vulnerability.yaml [hunk 8]
```diff
@@ -887,7 +887,7 @@
           V3Vector: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
       CweIDs:
       - CWE-502
-      Description: A deserialization of untrusted data vulnernerability exists in rails < 5.2.4.3, rails < 6.0.3.1 that can allow an attacker to unmarshal user-provided objects in MemCacheStore and RedisCacheStore potentially resulting in an RCE.
+      Description: A deserialization of untrusted data vulnerability exists in rails < 5.2.4.3, rails < 6.0.3.1 that can allow an attacker to unmarshal user-provided objects in MemCacheStore and RedisCacheStore potentially resulting in an RCE.
```
Comment (jsoref) [inline integration/testdata/fixtures/db/vulnerability.yaml:890]: These CVEs have errors in them. I'm happy to drop the files. If someone knows offhand how to get the CVEs fixed, that'd probably be better for the community overall.

File: integration/testdata/fixtures/db/vulnerability.yaml [hunk 9]
```diff
@@ -998,7 +998,7 @@
...
...e NUL terminated, even though this is not guaranteed for strings that have been directly constructed. Where an application requests an ASN.1 structure to be printed, and where that ASN.1 structure contains ASN1_STRINGs that have been directly constructed by the application without NUL terminating the "data" field, then a read buffer overrun can occur. The same thing can also occur during name constraints processing of certificates (for example if a certificate has been directly constructed by the application instead of loading it via the OpenSSL parsing functions, and the certificate contains non NUL terminated ASN1_STRING structures). It can also occur in the X509_get1_email(), X509_REQ_get1_email() and X509_get1_ocsp() functions. If a malicious actor can cause an application to directly construct an ASN1_STRING and then process it through one of the affected OpenSSL functions then this issue could be hit. This might result in a crash (causing a Denial of Service attack). It could also result in the disclosure of private memory contents (such as private keys, or sensitive plaintext). Fixed in OpenSSL 1.1.1l (Affected 1.1.1-1.1.1k). Fixed in OpenSSL 1.0.2za (Affected 1.0.2-1.0.2y).
```
Comment (jsoref) [inline integration/testdata/fixtures/db/vulnerability.yaml:1001]: These CVEs have errors in them. I'm happy to drop the files. If someone knows offhand how to get the CVEs fixed, that'd probably be better for the community overall.

File: pkg/rpc/client/client_test.go [hunk 10]
```diff
@@ -71,7 +71,7 @@ func TestScanner_Scan(t *testing.T) {
 								Title:            "DoS",
 								Description:      "Denial os Service",
 								Severity:         common.Severity_CRITICAL,
-								References:       []string{"http://exammple.com"},
+								References:       []string{"http://example.com"},
```
Comment (jsoref) [inline pkg/rpc/client/client_test.go:74]: ?

File: pkg/rpc/client/client_test.go [hunk 11]
```diff
@@ -71,7 +71,7 @@ func TestScanner_Scan(t *testing.T) {
 								Title:            "DoS",
 								Description:      "Denial os Service",
```
Comment (jsoref) [inline pkg/rpc/client/client_test.go:72]: I suspect this should be:
```suggestion
								Description:      "Denial of Service",
```

My tooling only looks at 3+ letter words by default (2 letter words had too much noise).

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets.

All fault is mine.

I'm happy to drop any particular change or changes to any particular sets of files.

**Review by chen-keinan — APPROVED** [review]

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**